### PR TITLE
Relicense.

### PR DESCRIPTION
**Pull request by danielballan**
All contributors should publicly sign off before this is merged.

### CODE REVIEW

**Comment by x94carbone** [comment]
Approved.

**Comment by kleinhenz** [comment]
Approved

File: LICENSE [hunk 1]
```diff
@@ -1,69 +1,29 @@
...
-   STATES, THE UNITED STATES DEPARTMENT OF ENERGY, OR THEIR EMPLOYEES BE LIABLE
-   FOR ANY DIRECT, INDIRECT, INCIDENTAL, CONSEQUENTIAL, SPECIAL OR PUNITIVE
-   DAMAGES OF ANY KIND OR NATURE RESULTING FROM EXERCISE OF THIS LICENSE
-   AGREEMENT OR THE USE OF THE SOFTWARE.
-
-END OF LICENSE
-
+BSD 3-Clause License
+
+Copyright (c) 2021, Brookhaven Science Associates, LLC, Brookhaven National Laboratory
```
Comment (dylanmcreynolds) [inline LICENSE:3]: 2022?

**Comment by x94carbone** [comment]
@danielballan @dylanmcreynolds I'd like to merge this if nobody has any objections. Just trying to cleanup old PRs.

**Comment by danielballan** [comment]
I confirmed with @elistavitski that the Funding Acknowledgement is correct, on 5 October, but did not yet find a moment to circle back here.

**Pull request merged**